src/components/MoneyReportHeader.tsx

```diff
@@ -185,7 +197,7 @@ function MoneyReportHeader({session, policy, chatReport, nextStep, report: money
                 shouldShowPinButton={false}
                 report={moneyRequestReport}
                 policy={policy}
-                shouldShowBackButton={isSmallScreenWidth}
+                shouldShowBackButton={shouldUseNarrowLayout || isSmallScreenWidth}
```

Comment:
I understand that in general the name `shouldUseNarrowLayout` is fitting, but I wanted to suggest a change.

When I look at this line it's clear to me that `isNarrowLayout` would be a better name.
In here, right next to each other we have `shouldXXX || isYYY` but both refer to the context of the screen/container and its sizing.
So I think if we're passing down layout related flags the names with `isXXX` are better and simpler.